src/GUI/RevEng.Core.50/RevEng.Core.50.csproj

```diff
@@ -33,7 +33,8 @@
 		<Compile Include="..\RevEng.Core\ServiceProviderBuilder.cs" Link="ServiceProviderBuilder.cs" />
 		<Compile Include="..\RevEng.Core\SharedTypeExtensions.cs" Link="SharedTypeExtensions.cs" />
 		<Compile Include="..\RevEng.Core\SqlServerHelper.cs" Link="SqlServerHelper.cs" />
-		<Compile Include="..\RevEng.Core\TableListBuilder.cs" Link="TableListBuilder.cs" />
+    <Compile Include="..\RevEng.Core\TableListBuilder.cs" Link="TableListBuilder.cs" />
```

Comment:
nit: Indent

Reply:
It is very strange, I don't see this indenting issue locally.

Reply:
Never mind..